$FreeBSD$

Stops clobbering CC, CFLAGS, LDFLAGS, forcing -O2 and adds
-lxcb-property to LDFLAGS to fix GCC46 build

http://www.freebsd.org/cgi/query-pr.cgi?pr=ports/163667

--- Makefile.orig	2011-12-28 14:56:20.865676731 +0000
+++ Makefile	2011-12-28 14:58:52.600677341 +0000
@@ -3,13 +3,11 @@
 SRC=mcwm.c list.c config.h events.h list.h
 DISTFILES=LICENSE Makefile NEWS README TODO WISHLIST mcwm.man $(SRC)
 
-CC=gcc
-CFLAGS=-g -std=c99 -Wall -Wextra -O2 -I/usr/local/include #-DDEBUG #-DDMALLOC
-LDFLAGS=-L/usr/local/lib -lxcb -lxcb-randr -lxcb-keysyms -lxcb-icccm \
-	-lxcb-atom #-ldmalloc
+CFLAGS+=-g -std=c99 -Wall -Wextra -I/usr/local/include #-DDEBUG #-DDMALLOC
+LDFLAGS+=-L/usr/local/lib -lxcb -lxcb-randr -lxcb-keysyms -lxcb-icccm \
+	-lxcb-atom -lxcb-property #-ldmalloc
 
 RM=/bin/rm
-PREFIX=/usr/local
 
 TARGETS=mcwm
 OBJS=mcwm.o list.o
